modules/magic_enum/0.9.5/presubmit.yml

```diff
@@ -0,0 +1,35 @@
+matrix:
+  platform:
+  - ubuntu2004
+  - macos
+  - macos_arm64
+  - windows
+  bazel:
+  - 6.x
```

Comment:
Is this intentional? Most Bzlmod users will probably be on 7.

Reply:
yes only for this version as the patches for the `.bazelrc` were very difficult due to bad line endings. https://github.com/bazelbuild/bazel-central-registry/pull/1903#issuecomment-2145510917

it has been fixed upstream and `7.x` can be added once again in the next magic_enum release. https://github.com/Neargye/magic_enum/pull/358